src/wp-includes/functions.php

```diff
@@ -3181,7 +3181,7 @@ function wp_check_filetype_and_ext( $file, $filename, $mimes = null ) {
 				$type = false;
 				$ext  = false;
 			}
-		} elseif ( 'application/csv' === $real_mime ) {
+		} elseif ( ( 'application/csv' === $real_mime ) || ( 'text/csv' === $real_mime ) ) {
```

Comment:
```suggestion
		} elseif ( 'application/csv' === $real_mime || 'text/csv' === $real_mime ) {
```
IMO, it looks a little cleaner to remove the additional parentheses here.